### PR TITLE
Admin

### PR DESCRIPTION
**Pull request by dgrosenblatt**
As as admin
I want to view a list of users
So I can see who is using the website

An admin 
I want to delete users, items, and reviews at my own discretion
So I can maintain some sort of control and decorum on the website

### CODE REVIEW

File: db/migrate/20150105194754_add_role_to_users.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class AddRoleToUsers < ActiveRecord::Migration
+  def change
+    add_column :users, :role, :string
```
Comment (radavis) [inline db/migrate/20150105194754_add_role_to_users.rb]: You will probably want to specify a default value for this field.

File: spec/features/admin_view_list_of_users_spec.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+feature "Admin views a list of users" do
+  scenario "while logged in as admin" do
+    @user = FactoryGirl.create(:user, role: "admin")
+    FactoryGirl.create_list(:user, 5)
+    login_as(@user)
+    visit "/admin/users"
```
Comment (radavis) [inline spec/features/admin_view_list_of_users_spec.rb]: Use the path helper.

File: spec/features/delete_review_spec.rb [hunk 3]
```diff
@@ -5,20 +5,18 @@
...
-    login_as(@user)
-  end
 
   scenario "clicking delete button" do
-    review = FactoryGirl.create(:review)
-    visit "/weathers/#{review.weather_id}"
+    @review = FactoryGirl.create(:review)
+    @user = @review.user
+    login_as(@user)
+    visit "/weathers/#{@review.weather_id}"
```
Comment (radavis) [inline spec/features/delete_review_spec.rb]: Use the path helper

File: spec/features/delete_review_spec.rb [hunk 4]
```diff
@@ -5,20 +5,18 @@
...
   } do
-  before :each do
-    @user = FactoryGirl.create(:user)
-    login_as(@user)
-  end
 
   scenario "clicking delete button" do
-    review = FactoryGirl.create(:review)
-    visit "/weathers/#{review.weather_id}"
+    @review = FactoryGirl.create(:review)
```
Comment (radavis) [inline spec/features/delete_review_spec.rb]: I would avoid using instance variables here.

**Comment by radavis** [comment]
Just a couple of things:
* Supply a default value for role on the users table. Your factory is using a default of "member". I would use that. Also validate that the role can only be 'admin' or 'member'.
* Whenever possible, use the path helpers specified by `rake routes`. 
* Only use instance variables when necessary. There are a few tests that have them and don't need them.

**Comment by dgrosenblatt** [comment]
Addressing comments from Richard

**Pull request merged**